packages/arb-tx-aggregator/batcher/batcher.go

```diff
@@ -236,6 +241,8 @@ func (m *Batcher) sendBatch(ctx context.Context, inbox arbbridge.GlobalInboxSend
 		return
 	}
 
+	m.newTxFeed.Send(core.NewTxsEvent{Txs: txes})
```

Comment:
I think the send to this feed should be in the send transaction function rather than here since we want to send to the newTxFeed immediately when we see a transaction and it might be a while before we send a batch